code/game/objects/structures/barsign.dm

```diff
@@ -186,7 +186,7 @@ var/list/barsigns = list()
 /obj/structure/sign/double/barsign/Topic(href, href_list)
 	if(..())
 		return
-	if(in_range(src, usr) && isliving(usr) || isAdminGhost(usr))
+	if(in_range(src, usr) && isliving(usr) || isAdminGhost(usr) || isAI(usr))
```

Comment:
probably want another allowed check here for sanity (unless it's covered at a higher level?)

mice are living

Reply:
Should be covered in attack_hand as thats the only way to get into the menu but another safety isn't bad